src/mock/task.js

```diff
@@ -0,0 +1,104 @@
+// рейтинг в профиле
+const RATINGNAME = [`Movie Buff`,
```

Comment:
```suggestion
const RATINGNAME = [
  `Movie Buff`,
```
Или в одну строку или так ))